frontend/pages/hosts/details/cards/Software/SelfService/SelfServiceItem/SelfServiceItem.tsx

```diff
@@ -37,6 +54,11 @@ const STATUS_CONFIG: Record<SoftwareInstallStatus, IStatusDisplayConfig> = {
     displayText: "Install in progress...",
     tooltip: () => "Software installation in progress...",
   },
+  blocked: {
+    iconName: "pending-outline", // TODO: add "disable" to type and change to disable icon
+    displayText: "TODO SelfServieItem.tsx", // TODO
```

Comment:
I don't think it's a good idea to leave TODOs in actual UI copy, easy to miss. What's left to do here?

Reply:
Actually, now looking at it, your ticket for the host details page is for modals only. This is missing in the spec.

Reply:
I updated my ticket title to include this and adding it in.